en/lessons/gravity-model.md

```diff
@@ -0,0 +1,807 @@
+
+**Calculating $s_{y}$ & $s_{x}$ (Standard Deviation)**
+
+[Standard deviation](https://en.wikipedia.org/wiki/Standard_deviation) is a way of expressing how much variation from the mean (average) there is in the data. In other words, is the data fairly clustered around the mean, or is the spread much wider?
+
+Again, there are online calculators and statistical software packages that can do this calculation for you if you provide the data.
+
+With the above values, you can calculate $β_{1}$. This will have to be done once for each of the five variables $β_{1}$ to $β_{5}$. These values allow you to calculate the y-intercept, $β_{0}$
+
+**Calculating $β_{0}$ (the y-Intercept)**
+
+Next, we have to calculate the y-intercept. The formula for calculating the y-intercept in a Simple Linear Regression is:
+
+$$β_{0} = \bar{y} - β \times \bar{x}$$
+
+However, the calculation becomes much more complicated in a multiple regression analysis, as each variable influences the calculation. This makes doing it by hand very difficult, and is one of the reasons we opt for a programmatic solution.
+
+### The Code for Calculating the Weightings
+
+The *MASS* statistical package, written for the *R* programming language, has a function that can solve negative binomial regression equations, making it very easy to calculate what would otherwise be a very difficult long-hand formula.
+
+This section assumes you have installed *R* and have installed the *MASS* package. If you have not done so you will have to before proceeding. Taryn Dewar's tutorial on [R Basics with Tabular Data](/en/lessons/r-basics-with-tabular-data) includes *R* installation instructions.
+
+To use this code, you will need to download a copy of the dataset of the five variables plus the number of observed vagrants from each of the 32 counties. This is available above as Table 3, or can be downloaded as a [.csv file](/assets/gravity-model/VagrantsExampleData.csv). Whatever mode you choose, save the file as *VagrantsExampleData.csv*. If you are using a Mac make sure you save it as a [Windows format .csv](https://superuser.com/questions/385265/whats-the-difference-between-csv-ms-dos-csv-macintosh-csv-comma-delimi) file. Open *VagrantsExampleData.csv* and familiarise yourself with its contents. You should notice each of the 32 counties, along with each of the variables we've discussed throughout this tutorial. We will be using the column headers to access this data with our computer programme. I could have called them anything, but in this file they are:
+
+1. vagrants
```

Comment:
As I requested above, all names that correspond to table or R variables, functions, expressions should be marked with backticks.

Reply:
I've added back ticks to variables. I don't think there's internal consistency here on what needs backticks and what needs Latex formatting. But that's why we need a style sheet!